themes/default/content/registry/packages/harbor/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Harbor
 meta_desc: Provides an overview of the Harbor Provider for Pulumi.
-layout: package
+layout: overview
```

Comment:
@dirien can you update this to `package` in your harbor `docs` folder for both Markdown files?

Reply:
This does need to be updated, but we relaxed the requirement on this last week, so it can be done when time allows. Thank you both! (I'll go ahead and merge this now.)